src/arrayReverse.test.js

```diff
@@ -8,13 +8,62 @@ describe(`Function 'arrayReverse':`, () => {
   });
 
   it(`should return an array`, () => {
+    expect(arrayReverse([''])).toBeInstanceOf(Array);
+  });
 
+  it('should return an empty string '
+    + 'if original array consists of an empty string', () => {
+    expect(arrayReverse(['', '']))
+      .toEqual(['', '']);
   });
 
-  it(`should return an empty string
-    if original array consists of an empty string`, () => {
-    expect(arrayReverse(['Mate', 'Academy'])).toEqual(['ymed', 'acAetaM']);
+  it('should correctly work with long strings', () => {
+    const StringArray = [
+      'a',
+      'bC',
+      'dEf',
+      'gHiJ',
+      'kLmNo',
+      'pQrStU',
+      'vWxYz',
+      'aBcD',
+      'eFg',
+      'hI',
+    ];
+
+    const ExpectedResult = [
+      'I',
+      'hg',
+      'FeD',
+      'cBaz',
+      'YxWvU',
+      'tSrQpo',
+      'NmLkJ',
+      'iHgf',
+      'EdC',
+      'ba',
+    ];
+
+    expect(arrayReverse(StringArray))
+      .toEqual(ExpectedResult);
   });
 
-  // write more tests here
+  it('should ignore empty strings', () => {
+    const StringArray = [
+      '',
+      'Abc',
+      'De',
+      '',
+    ];
+
+    const ExpectedResult = [
+      '',
+      'eDc',
+      'bA',
+      '',
+    ];
+
+    expect(arrayReverse(StringArray))
+      .toEqual(ExpectedResult);
+  });
```

Comment:
add more tests. Check the string with only one character, with numbers, with special symbols; check array with only one string